exercises/concept/lucians-luscious-lasagna/LuciansLusciousLasagnaTests.cs

```diff
@@ -12,11 +12,18 @@ public void Expected_minutes_in_oven()
 
     [Fact]
     [Task(2)]
-    public void Remaining_minutes_in_oven()
+    public void Remaining_minutes_in_oven_for_fifteen()
```

Comment:
My only tiny nit is that I think I would prefer the test name to be about the input, so:

```suggestion
    public void Remaining_minutes_in_oven_after_twenty_five_minutes()
```